scss/tech.scss

```diff
@@ -0,0 +1,54 @@
+.tech {
+  padding: 10px 40px;
+}
+
+.tech-container {
+  align-items: center;
+  background-color: $green;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  flex-basis: 150px;
+  left: 10px;
+  margin-bottom: 50px;
```

Comment:
when expressing a list of elements it's always better to use the same margin used on the bottom on the top and the same with left and right. this leaves the elements perfectly centered.